modules/i3s/src/lib/helpers/i3s-nodepages-tiles.ts

```diff
@@ -77,7 +77,7 @@ export default class I3SNodePagesTiles {
     let contentUrl = null;
     let textureUrl: string | null = null;
     let materialDefinition = null;
-    let textureFormat = 'jpeg';
+    let textureFormat = 'jpg';
```

Comment:
Why the change? Seems unrelated?

Reply:
In spec - https://github.com/Esri/i3s-spec/blob/master/docs/1.8/textureSetDefinitionFormat.cmn.md no "jpeg" texture format, only "jpg". Because of this, in the getLoaderForTextureFormat function, instead of choosing a loader, the default loader was selected.